jbmc/regression/jbmc/exceptions28/test.desc

```diff
@@ -0,0 +1,17 @@
+KNOWNBUG
+test.class
+--unwind 10
+^\[java::test.main:\(\[Ljava/lang/String;\)V\.1\] line 7 no uncaught exception: FAILURE$
+^VERIFICATION FAILED$
+^EXIT=10$
+^SIGNAL=0$
+--
+^warning: ignoring
+--
+This test verifies that we can handle the exception table not being ordered
```

Comment:
:question: What does "not being ordered" mean?

Reply:
Added extra comment below example.